indigo_app/notifications.py

```diff
@@ -51,3 +68,37 @@ def post_save_task(sender, instance, **kwargs):
     if kwargs['created']:
         if isinstance(instance.action_object, Task):
             notifier.send_task_notifications(instance)
+
+
+@receiver(comment_was_posted, sender=Comment)
+def post_comment_save_notification(sender, **kwargs):
+    """
+    Send email when a user comments on a task
+    """
+    task_content_type = ContentType.objects.get_for_model(Task)
+
+    notifier = Notifier()
```

Comment:
Simply use the existing `notifier` singleton instance, in the same way `post_save_task` works.